adba/aniDBAbstracter.py

```diff
@@ -196,13 +202,8 @@ def _get_name_from_xml(self, aid, onlyMain=True):
                         return title.text
         return ""
 
-    @staticmethod
-    def _read_animetitels_xml(path=None):
-        if not path:
-            path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "animetitles.xml")
-
-        f = open(path, "r")
-        all_anime_xml = etree.ElementTree(file=f)
+    def _read_animetitels_xml(self):
```

Comment:
This function can be removed altogether and `read_anidb_xml` used directly instead.